src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -4624,6 +4654,18 @@ theorem getLsbD_intMax (w : Nat) : (intMax w).getLsbD i = decide (i + 1 < w) :=
   · simp [h]
   · rw [Nat.sub_add_cancel (Nat.two_pow_pos (w - 1)), Nat.two_pow_pred_mod_two_pow (by omega)]
 
+@[simp]
+theorem toInt_intMax : (BitVec.intMax w).toInt = 2 ^ (w - 1) - 1 := by
```

Comment:
```suggestion
theorem toInt_intMax : (intMax w).toInt = 2 ^ (w - 1) - 1 := by
```

Since we are in the namespace, I don't think we need `BitVec.`? Please do double-check!